src/App.js

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
 
```

Comment:
Consider removing redundant new line.
You have two new lines, delete one.

Reply:
/fxd